spec/system/candidate_interface/candidate_signing_out_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'A candidate signing out' do
+  context 'when a candidate is signed in' do
+    before do
+      candidate = FactoryBot.create(:candidate)
+      login_as(candidate)
+
+      visit candidate_interface_welcome_path
+    end
+
+    it 'can see a sign out button' do
+      expect(page).to have_selector :link_or_button, 'Sign out'
+    end
+
+    context 'when the sign out button is clicked' do
```

Comment:
this sounds like an action, you could write:
` describe 'click sign out button' do`